README.md

```diff
@@ -4,10 +4,11 @@
 
 ### Why equity compensation matters
 
-**Equity compensation** is the practice of granting ownership in a company in exchange for
-work. In its ideal form, equity compensation aligns the interests of individual employees
-with the goals of the company they work for, which can yield dramatic results in team
-building, innovation, and longevity of employment.
+**Equity compensation** is the practice of granting some form of ownership in a company in
```

Comment:
partial ownership? (This is in a current commit I'm working on.) "some form" introduces too many questions at too early a point imo.

Reply:
Sure, will go with partial.